src/SAWScript/Panic.hs

```diff
@@ -0,0 +1,27 @@
+{- |
+Module      : SAWScript.Panic
+License     : BSD3
+Maintainer  : huffman
+Stability   : provisional
+-}
+
+{-# LANGUAGE Trustworthy, TemplateHaskell #-}
+module SAWScript.Panic
+  (HasCallStack, SawPanic, Saw, Panic, panic) where
+
+import Panic hiding (panic)
+import qualified Panic as Panic
+
+data Saw = Saw
+
+type SawPanic = Panic Saw
```

Comment:
why is this needed?

Reply:
I'm not sure; I copied it from cryptol. Maybe we don't need it.